modules/gapi/perf/common/gapi_video_perf_tests_inl.hpp

```diff
@@ -178,47 +178,53 @@ PERF_TEST_P_(BackgroundSubtractorPerfTest, TestPerformance)
         thr = 16.;
     else if (opType == gvideo::TYPE_BS_KNN)
         thr = 400.;
```

Comment:
Do you test case when thr = -1?

Reply:
No I don't. These values are the standard for `MOG2` and `KNN`, as they are used in our accuracy tests

Reply:
Assert after if, else if for -1 case. Someone can change tests in future.

Reply:
Actually, I agree it should be handled, but I wasn't sure. Thanks for insisting :)
Applied